src/main/sensors/esc_sensor.h

```diff
@@ -24,6 +24,8 @@
 
 typedef struct escSensorConfig_s {
     uint8_t halfDuplex;             // Set to false to listen on the TX pin for telemetry data
+    uint16_t offset;                // offset consumed by the flight controller / VTX / cam / ... in milliampere
```

Comment:
Maybe add a reserved `uint8_t` field above offset, or pack the struct explicitly.

Reply:
Structure field alignment is well defined, implicit padding should be fine. There is no need to do it explicitly.

Reply:
Without looking at the source, as far as I know there are issues with packing structs that are used in parameter groups.

Re adding explicit padding, I think that this will make it harder for the next person wanting to add an element to do the right thing (i.e. just append).

What we should do is that, whenever a change to a parameter group requiring a version increment is done, the rest of the struct should be scrutinised, and if necessary rearranged for optimum space use.

Reply:
It is a bug if there is problem with aligned structs. 
using `attribute ((packed))` may have undesired side effects, so it is not used.

Reordering fields may be done occasionally, but hopefully the PG management is automated before memory saving is significant